README.md

```diff
@@ -1,15 +1,52 @@
 # OSM Health
-... is a tool for troubleshooting issues with [Open Service Mesh](https://github.com/openservicemesh/osm).
+OSM Health is a tool for troubleshooting issues with [Open Service Mesh](https://github.com/openservicemesh/osm). It can
+be used to gather diagnostic information on the state of the control plane and the mesh components to ensure that 
+everything is running as expected.
 
 
 ## Get started
-Build the binary
 
-```
-make build-osm-health
+1. Clone the repo
+1. Build the binary
+
+    ```bash
+    make build-osm-health
+    ```
+
+1. Add it to `$PATH` to get started
+    ```bash
+    sudo cp ./bin/osm-health /usr/local/bin/osm-health
+    ```
+
+You can now use the CLI with:
+```bash
+osm-health <command>
 ```
 
-For testing, use the CLI with:
+## Commands
+Currently, osm-health checks the connectivity between two pods by running a series of diagnostic checks on the meshed namespaces and pods, 
+Envoy, SMI policies and core OSM control plane components. To run these checks, use:
+
+```bash
+osm-health connectivity pod-to-pod <SOURCE_POD> <DESTINATION_POD>
 ```
-bin/osm-health <command>
+
+For the most up to date list of available commands:
+```bash
+osm-health --help
```

Comment:
we currently have a number of commands listed by `osm-health` that have no checks yet. We should probably remove these from the help outcome?

Reply:
I think we should definitely remove the subcommands that are stubs - those will confuse users. However, let's merge this doc as it contains the core ideas of osm-health. Once we decide on a v0.1 release date + removed the stub subcommands, we can revisit this doc and fine-tune the info here.